bin/run-in-docker.sh

```diff
@@ -1,39 +1,42 @@
-#!/usr/bin/env bash
-set -e
```

Comment:
Why did you remove this? This was added explicitly to ensure that not being able to build fails the entire script, and not silently continues.

Reply:
I was following the [generic-test-runner](https://gihub.com/exercism/generic-test-runner) files which didn't include it.

Reply:
Oh, that's a mistake on our part then! Sorry about that.